lib/MediaWords/DBI/Stories/t/AP.t

```diff
@@ -0,0 +1,132 @@
+use strict;
```

Comment:
Thanks for the test, it will save me some opiate pills when rewriting this to Python!